index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <div id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p id ="title-description">The man who saved a billion lives</p>
+    <div id="img-div">
+      <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
```

Comment:
linters will give error in image name you should rename it

Reply:
- [ ]  Always make sure to use polite words like Kindly, It would be better etc. Kindly edit your comment and submit the project.